packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/location.tsp

```diff
@@ -0,0 +1,179 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.Models.Resources;
+
+@resource("locationTrackedResources")
+@parentResource(ArmLocationResource<"ResourceGroup">)
```

Comment:
Use `ResourceGroupLocationResource`

Reply:
@XiaofeiCao 
Replace `ArmLocationResource` with `ResourceGroupLocationResource` ?

Reply:
Yes. They should be equivalent, though cleaner.

Reply:
@XiaofeiCao 
The `ResourceGroupLocationResource` does not seem to be a valid template. An exception occurs when executing `pnpm pr-prep`. The exception information is as follows: `Can't pass template arguments to non-templated type.`

Reply:
`@parentResource(ResourceGroupLocationResource)`

Reply:
Fixed in the new version.